lib/src/util/rem_util.dart

```diff
@@ -20,12 +20,14 @@ import 'dart:async';
 import 'dart:html';
 
 import 'package:meta/meta.dart';
-import 'package:over_react/over_react.dart';
-import 'package:over_react/components.dart' as v2;
+import 'package:over_react/src/component/dom_components.dart';
+import 'package:over_react/src/component/resize_sensor.dart' as v2;
 import 'package:over_react/src/component_declaration/component_base.dart' as component_base;
-import 'package:over_react/react_dom.dart' as react_dom;
+import 'package:over_react/src/util/css_value_util.dart';
 import 'package:platform_detect/platform_detect.dart';
 
+import '../../react_dom.dart' as react_dom;
```

Comment:
Could we make this a `package:` import instead of a relative path?

Reply:
because that file is lib/react_dom.dart so I couldn't get a src import